pandas/core/base.py

```diff
@@ -801,7 +801,31 @@ def argmax(self, axis=None):
         return nanops.nanargmax(self.values)
 
     def min(self):
-        """ The minimum value of the object """
+        """
+        Return the minimum value of the Index object.
+
+        Return the minimum value of the object within the same type.
+        The Index of a DataFrame can be accessed as pandas.DataFrame.index.
+
+        Returns
+        -------
+        Scalar or object.
+            minimum value
```

Comment:
Sorry, perhaps I wasn't clear before.
```
scalar or object
            Minimum value.
```

I'll quote the docs:

> The documentation of the return is also similar to the parameters. But in this case, no name will be provided, unless the method returns or yields more than one value (a tuple of values).

> The parameters are defined by their name, followed by a space, a colon, another space, and the type (or types). Note that the space between the name and the colon is important. Types are not defined for *args and **kwargs, but must be defined for all other parameters. After the parameter definition, it is required to have a line with the parameter description, which is indented, and can have multiple lines. The description must start with a capital letter, and finish with a dot.

Reply:
I think now should be fine, thanks!